lib/KDropdownMenu.vue

```diff
@@ -147,6 +149,7 @@
       handleOpen() {
         this.$nextTick(() => this.$nextTick(() => this.setFocus()));
         window.addEventListener('keydown', this.handleOpenMenuNavigation, true);
+        this.lastFocusElement = document.activeElement;
```

Comment:
Although there is no problem with having this line as the last line of the `handleOpen` method, we can move it to be the first line of the method, so the intention of storing the last focused element before we focus anything else is clearer :).